ec2system/ec2machine.go

```diff
@@ -604,6 +610,9 @@ func (s *System) Start(ctx context.Context, count int) ([]*bigmachine.Machine, e
 		}
 		machines[i] = new(bigmachine.Machine)
 		machines[i].Addr = fmt.Sprintf("https://%s/", addr)
+		if useInstanceIDPrefix {
+			machines[i].Addr += "/" + aws.StringValue(instance.InstanceId) + "/"
```

Comment:
```suggestion
			machines[i].Addr += aws.StringValue(instance.InstanceId) + "/"
```
...otherwise you get a `//` which I think is legal but weird?